pyanaconda/modules/storage/storage.py

```diff
@@ -100,6 +101,9 @@ def __init__(self):
         self._nvdimm_module = NVDIMMModule()
         self._add_module(self._nvdimm_module)
 
+        self._nvme_module = NVMEModule()
+        self._modules.add_module(self._nvme_module)
```

Comment:
Fixed, I didn't notice the difference between `master` and `rhel-9` versions of the code when backporting. The tests would catch this, but they now fail before reaching this because of the libblockdev 3.0 issue :-)

Reply:
> because of the libblockdev 3.0 issue

Should be fixed by `python-blivet-3.6.0-11.el9`